src/webgpu/web_platform/util.ts

```diff
@@ -466,7 +466,9 @@ export async function getVideoFrameFromVideoElement(
 
   return raceWithRejectOnTimeout(
     new Promise<VideoFrame>(resolve => {
-      const videoTrack: MediaStreamVideoTrack = video.captureStream().getVideoTracks()[0];
+      const videoTrack: MediaStreamVideoTrack = video
+        .captureStream()
+        .getVideoTracks()[0] as MediaStreamVideoTrack;
```

Comment:
What else can the type be that requires a cast?

Reply:
It could be "MediaStreamTrack". Unfortunately, this type conflict is introduced by "imageCapture" type file (through reference webrtc here https://www.npmjs.com/package/@types/w3c-image-capture?activeTab=code). The conflicts are:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/webrtc/MediaStream.d.ts#L86 which return type is `MediaStreamTrack`
and https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/dom-mediacapture-transform/index.d.ts#L28 which return type is `MediaStreamVideoTrack`.